BepInEx.Core/Configuration/ConfigFile.cs

```diff
@@ -217,6 +229,63 @@ public ConfigEntryBase[] GetConfigEntries()
                 return Entries.Values.ToArray();
         }
 
+        /// <summary>
+        /// Get the value of an orphaned entry.
+        /// </summary>
+        /// <param name="entry">The entry to try and retrieve.</param>
+        /// <param name="val">Out value of this entry (if it exists).</param>
+        /// <returns>True if the entry exists in the dictionary.</returns>
+        public bool TryGetOrphanedEntry(ConfigDefinition entry, out string val)
+        {
+            lock (_ioLock)
+                return _orphanedEntries.TryGetValue(entry, out val);
+        }
+
+        /// <summary>
+        /// Get the value of an orphaned entry.
+        /// </summary>
+        /// <param name="section">Section of the orphaned entry.</param>
+        /// <param name="key">Key of the orphaned entry.</param>
+        /// <param name="val">Out value of this entry (if it exists).</param>
+        /// <returns>True if the entry exists in the dictionary.</returns>
+        public bool TryGetOrphanedEntry(string section, string key, out string val)
+        {
+            lock (_ioLock)
+                return _orphanedEntries.TryGetValue(new ConfigDefinition(section, key), out val);
+        }
+
+        /// <summary>
+        /// Removes an orphaned config entry from the file.
+        /// </summary>
+        /// <param name="entry">Orphaned config entry to remove.</param>
+        /// <returns>True on success.</returns>
+        public bool TryRemoveOrphanedEntry(ConfigDefinition entry)
+        {
+            lock (_ioLock)
+                return _orphanedEntries.Remove(entry);
+        }
+
+        /// <summary>
+        /// Removes an orphaned config entry from the file.
+        /// </summary>
+        /// <param name="section">Section of orphaned entry to remove.</param>
```

Comment:
There's a word missing in `Section of *the* orphaned entry to remove.` (to stay consistent with the wording in the rest of the xmldoc)
Same thing on the line below.

Reply:
56655305fa40a04494c75d60ed75967c3df3ac22